tools/StaticAnalysis/Exceptions/Az.SignalR/ExtraAssemblies.csv

```diff
@@ -1 +1,6 @@
 "Directory","AssemblyName","Severity","ProblemId","Description","Remediation"
+"src\Package\Debug\ResourceManager\AzureResourceManager\Azure.Storage","Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms","2","2000","Assembly Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms is not referenced from any cmdlets assembly","Remove assembly Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms from the project and regenerate the Wix file"
+"src\Package\Debug\ResourceManager\AzureResourceManager\Azure.Storage","Microsoft.Threading.Tasks.Extensions.Desktop","2","2000","Assembly Microsoft.Threading.Tasks.Extensions.Desktop is not referenced from any cmdlets assembly","Remove assembly Microsoft.Threading.Tasks.Extensions.Desktop from the project and regenerate the Wix file"
+"src\Package\Debug\ResourceManager\AzureResourceManager\Azure.Storage","Microsoft.Threading.Tasks.Extensions","2","2000","Assembly Microsoft.Threading.Tasks.Extensions is not referenced from any cmdlets assembly","Remove assembly Microsoft.Threading.Tasks.Extensions from the project and regenerate the Wix file"
+"src\Package\Debug\ResourceManager\AzureResourceManager\Azure.Storage","System.Net.Http.Extensions","2","2000","Assembly System.Net.Http.Extensions is not referenced from any cmdlets assembly","Remove assembly System.Net.Http.Extensions from the project and regenerate the Wix file"
+"src\Package\Debug\ResourceManager\AzureResourceManager\Azure.Storage","System.Net.Http.Primitives","2","2000","Assembly System.Net.Http.Primitives is not referenced from any cmdlets assembly","Remove assembly System.Net.Http.Primitives from the project and regenerate the Wix file"
```

Comment:
It seems this file should be under folder Az.Storage instead of Az.SignalR

Reply:
I just compared tools folder, and add the missing chnage.
This is from maddieclayton in commit https://github.com/Azure/azure-powershell/commit/4344a9851a8237fcdbb47cf2e6b15f7a4d997c99

Anyway, the path not exist any more, I can try to remove it.
But if remove it cause any issue, should wee it back.